index.html

```diff
@@ -1514,27 +1514,47 @@ <h2 id="docconformance">
             </td>
           </tr>
           <tr>
-            <th id="el-input-image" tabindex="-1">
-              <a data-cite="html/input.html#image-button-state-(type=image)">`input type=image`</a>
+            <th id="el-button-input-image-reset-submit-form" tabindex="-1">
+             [^button^] with a missing or invalid [^button/type^], <a data-cite="html/input.html#image-button-state-(type=image)">`input type=image`</a>, <a data-cite="html/input.html#submit-button-state-(type=submit)">`input type=submit`</a>, and <a data-cite="html/input.html#reset-button-state-(type=reset)">`input type=reset`</a> , if a descendant of a [^form^] element.
```

Comment:
Should be fixed as the following:
- Before fixed：[^button^] with a missing or invalid [^button/type^], 
- After fixed： \`button type=submit\`, \`button type=reset\`, and [^button^] with a missing or invalid [^button/type^], 

Because in functionaly `button type=submit` is equal `input type=submit`, and `button type=reset` is equal `input type=reset`.